imports/ui/unit-explorer/unit-type-icon.jsx

```diff
@@ -52,5 +52,6 @@ export default class UnitTypeIcon extends Component {
 
 UnitTypeIcon.propTypes = {
   metaData: PropTypes.object,
-  iconInExplorer: PropTypes.string
+  iconInExplorer: PropTypes.string,
+  unitType: PropTypes.string
```

Comment:
I can't find any usages of this new prop. Please remove it if it's not going to be used